packages/account/test/fuel-wallet-connector.test.ts

```diff
@@ -557,49 +553,16 @@ describe('Fuel Connector', () => {
       wallets: [defaultWallet],
     });
 
-    let hasWallet = await new Fuel().hasWallet();
+    let hasWallet = await (await new Fuel().init()).hasWallet();
 
     expect(hasWallet).toBeFalsy();
 
-    hasWallet = await new Fuel({
-      connectors: [connector],
-    }).hasWallet();
+    hasWallet = await (
+      await new Fuel({
+        connectors: [connector],
+      }).init()
+    ).hasWallet();
 
     expect(hasWallet).toBeTruthy();
   });
-
-  it('should ensure getProvider works just fine', async () => {
```

Comment:
`getProvider` has been deprecated for some time now and so we no longer need to test this method.

Reply:
Has the method been removed or just deprecated?

In short, I think the test should be removed only when the method is removed.

Otherwise, people may still be using it, in which case the tests are still valid.

Reply:
My perspective is that if it's deprecated we no longer support or maintain it (including running tests against it), so if some experiences an issue with it with should deter them from using it. But I'm happy to leave the tests here and we can address that on another issue if needs be.

Reply:
Deprecated methods are still supported, which makes the test deletion potentially dangerous.

If we remove the test, we can introduce some changes that break it silently, and then if we publish a `patch`, which should be backward compatible, and users update to it, it may break their apps.